about.html

```diff
@@ -0,0 +1,172 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>About RMsoft</title>
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.3.0/css/font-awesome.min.css"
+    />
+    <link
+      href="https://fonts.googleapis.com/css?family=Lato"
+      rel="stylesheet"
+    />
+    <link href="https://fonts.cdnfonts.com/css/cocogoose" rel="stylesheet" />
+    <link rel="stylesheet" href="css/all.css" />
+    <link rel="stylesheet" href="css/about.css" />
+    <link
+      rel="stylesheet"
+      media="screen and (min-width: 768px)"
+      href="css/widescreen.css"
+    />
+    <script src="scripts.js" defer></script>
+  </head>
+  <body>
+    <main>
+      <header>
+        <!-- Mobile nav -->
+        <div class="hamburger">
+          <div></div>
+          <div></div>
+          <div></div>
+        </div>
+        <!-- mobile nav -->
+        <nav id="clicked" class="not-cliked fonts">
+          <p class="clicked-p">X</p>
+          <ul class="clicked-ul">
+            <li>
+              <a href="index.html">Home</a>
+            </li>
+            <li>
+              <a href="about.html" class="active">About</a>
+            </li>
+            <li>
+              <a href="#">Projects</a>
+            </li>
+            <li>
+              <a href="#">Services</a>
+            </li>
```

Comment:
- [ ] Kindly ensure that on the mobile version, the nav menu span across all the height of the page. 🙂, this makes your design look very beautiful.

Please, check [this](https://stackoverflow.com/questions/20018742/make-navigation-bar-take-up-entire-page-height-in-css) stackoverflow thread for more 😉.

![mobile](https://user-images.githubusercontent.com/41163571/225405985-073a0aa6-0aee-46bd-8c83-050ce9f46d0e.gif)